bin/dependency_validator.dart

```diff
@@ -16,11 +16,13 @@ import 'dart:io' show exit, stderr, stdout;
 
 import 'package:args/args.dart';
 import 'package:dependency_validator/dependency_validator.dart';
+import 'package:io/io.dart';
```

Comment:
This isn't a dependency in v2, so we need to add it:

```yaml
  io: ^0.3.5
```

Reply:
done